common/workunit/workunit.cpp

```diff
@@ -6166,9 +6150,8 @@ void CLocalWorkUnit::releaseFile(const char *fileName)
     }
 }
 
-void CLocalWorkUnit::clearGraphProgress()
+void CLocalWorkUnit::clearGraphProgress() const
```

Comment:
Should this really be const ?
I know it's doesn't alter the contents of CLocalWorkUnit, but the progress is logically part of it.
The only reason it wasn't nested like other parts of it was to do with locking and lazy fetching.